group_trigger.go

```diff
@@ -81,22 +82,22 @@ func (tc *TriggerCounter) countNewTriggers(tList *triggerList) error {
 	tc.keyFrame = tList.keyFrame
 	tc.keyTime = tList.keyTime
 	tc.sampleRate = tList.sampleRate
+	if tc.sampleRate <= 0 {
```

Comment:
Why return nil here? shouldnt negative sample rate be indicative of an error or some sort?

Reply:
This fixed some strange error I was having that arose from messages being generated in a triangle or other fake-data sample, where there was no sample rate. I just decided that having a zero sample rate was not a problem that needed to be diagnosed (or panicked at) here in the group trigger.